modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -8,103 +8,37 @@
 
 You can configure your hub cluster with a set of ArgoCD applications that generate the required installation and policy custom resources (CR) for each site based on a zero touch provisioning (ZTP) GitOps flow.
 
-.Procedure
+.Prerequisites
 
-. Install the Red Hat OpenShift GitOps Operator on your hub cluster.
+* Openshift Cluster 4.8 or 4.9 as the hub cluster
+* {rh-rhacm-first} Operator 2.3 or 2.4 installed on the hub cluster
+* Red Hat OpenShift GitOps Operator 1.3 on the hub cluster
 
-. Extract the administrator password for ArgoCD:
-+
-[source,terminal]
-----
-$ oc get secret openshift-gitops-cluster -n openshift-gitops -o jsonpath='{.data.admin\.password}' | base64 -d
-----
+.Procedure
+
+. Install the {cgu-operatpor-first}, which coordinates with any new sites added by ZTP and manages application of the `PolicyGenTemplate`-generated policies.
```

Comment:
"{cgu-operatpor-first}" should be changed. Not sure what  to change to exactly, but it's currently shown as is in the html doc.